maps/southern_cross/items/headset_sc.dm

```diff
@@ -1,32 +1,30 @@
 /obj/item/device/radio/headset/pilot
 	name = "pilot's headset"
-	desc = "A headset used by pilots, has access to supply and explorer channels."
+	desc = "A headset used by pilots, has access to explorer channel." //VOREStation Edit
 	icon_state = "pilot_headset"
 	adhoc_fallback = TRUE
 	ks2type = /obj/item/device/encryptionkey/pilot
-	adhoc_fallback = TRUE //VOREStation Edit
```

Comment:
What did this do and why was it removed?

Reply:
From the looks of it, at some point polaris did not have adhoc_fallback on those headsets, those were edited into the Vorestation PR, but then polaris added it in at some point without Vorestation's edit being removed, so it ends up being the same line duplicated twice.

Reply:
That is the case, yes. I added the feature and added it on headsets here, but didn't want to decide for them whether they wanted it. They eventually added it.